src/Rule/Rule.php

```diff
@@ -220,30 +215,6 @@ public function addValue($value, $type = ' '): void
         }
     }
 
-    /**
-     * @param int $modifier
-     */
-    public function addIeHack($modifier): void
-    {
-        $this->ieHack[] = $modifier;
-    }
```

Comment:
In #993, this wasn't deprecated.  This should be done first.

Reply:
Done: #1003